src/Raven.Server/Config/PersistConfigurationException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Raven.Server.Config;
```

Comment:
move to Raven.Server.Exceptions

Reply:
should this inherit from RavenException?

Reply:
> Raven.Server.Exceptions

done

Reply:
> should this inherit from RavenException?

done